topdown/http_test.go

```diff
@@ -220,3 +220,69 @@ func TestInvalidKeyError(t *testing.T) {
 		runTopDownTestCase(t, data, tc.note, tc.rules, tc.expected)
 	}
 }
+
+// TestHTTPRedirectDisable tests redirects are not enabled by default
+func TestHTTPRedirectDisable(t *testing.T) {
+
+	// expected result
+	expectedResult := make(map[string]interface{})
+	expectedResult["status"] = "301 Moved Permanently"
+	expectedResult["status_code"] = http.StatusMovedPermanently
+	expectedResult["body"] = nil
+
+	resultObj, err := ast.InterfaceToValue(expectedResult)
+	if err != nil {
+		panic(err)
+	}
+
+	var testURL = "http://google.com"
```

Comment:
💡 That's clever!  We could also use https://nghttp2.org/httpbin/, which provides different kinds of redirects:
```
/redirect/:n 302 Redirects n times.
/redirect-to?url=foo 302 Redirects to the foo URL.
/relative-redirect/:n 302 Relative redirects n times.
/absolute-redirect/:n 302 Absolute redirects n times.
```